.github/workflows/main.yml

```diff
@@ -1997,6 +2000,12 @@ jobs:
       - name: tomcat5
         run: ant $OPTS -f enterprise/tomcat5 test
 
+      - name: enterprise/j2ee.core
+        run: ant $OPTS -f enterprise/web.jsf test
+
+      - name: enterprise/j2ee.core
+        run: ant $OPTS -f enterprise/web.jsfapi test
```

Comment:
same here, please update names to `web.jsf` and `web.jsfapi`

Reply:
Oups. Done!